README.md

```diff
@@ -181,6 +181,22 @@ and [this video](https://www.youtube.com/watch?v=T0dDetqgra4&ab_channel=Rasa) ca
 
 This notebook allows you to use embeddings and a drawing tool to do some bulk-labelling.
 
+## Running with Docker Compose
+
+You can run all of the above commands at the same time and access them all from the same URL using the provided `docker-compose.yaml`.
+
+To do this, [make sure Docker Compose is installed](https://docs.docker.com/compose/install/) and run:
+
+```bash
+# If you haven't already cloned this repo.
+git clone git@github.com:RasaHQ/rasalit.git
+cd rasalit
+
+RASA_PROJECT_DIR=/path/to/my_rasa_project OVERVIEW_FOLDER=/path/to/gridresults docker-compose up -d
```

Comment:
This could be simplified if there were a GitHub action to build all of these containers and push them up to DockerHub, in which case you wouldn't need the whole Rasalit repository and to build the containers yourself (which takes a while). In this case, you would only need to curl the `docker-compose.yaml` file from raw.githubusercontent.com and pull the pre-built containers from DockerHub (e.g. `RasaHQ/rasalit-overview`, `RasaHQ/rasalit-spelling`, etc.).